www/routes/index.tsx

```diff
@@ -151,33 +188,35 @@ function Intro() {
 function GettingStarted(props: { origin: string }) {
   return (
     <section
-      class={tw`max-w-screen-sm mx-auto my-16 px(4 sm:6 md:8) space-y-4`}
+      class={tw`max-w-screen-md mx-auto my-16 px(4 sm:6 md:8) space-y-4`}
     >
-      <h2 id="getting-started" class={tw`text(xl gray-600) font-bold`}>
+      <h2 id="getting-started" class={tw`text(3xl gray-600) font-bold`}>
         <a href="#getting-started" class={tw`hover:underline`}>
-          Getting started
+          Getting Started
         </a>
       </h2>
-      <p class={tw`text-gray-600`}>
-        To get started, make sure you have the{" "}
+      <p class={tw`text-gray-600 flex gap-1 mb-4 bg-gray-100 p-2 rounded`}>
+        <span class={tw`text-gray-400`}>
+          <Icons.Info />
+        </span>
         <a href="https://deno.land" class={tw`text-blue-600 hover:underline`}>
           Deno CLI
         </a>{" "}
-        version 1.23.0 or higher installed.
+        version 1.23.0 or higher required.
```

Comment:
```suggestion
        version 1.23.0 or higher is required.
```

Can you also add a link to where to install / upgrade Deno?